lib/Components/CollectionComponent.ts

```diff
@@ -85,17 +100,17 @@ export class CollectionComponent extends Component {
      *
      * @private
      */
-    _prepareArrayContent() {
-        return this._defaultPrepare(this.collection.map(model => this.prepareChild(model)));
+    protected _prepareArrayContent() {
+        return this._defaultPrepare((this.collection as Object[]).map(model => this.prepareChild(model)));
     }
 
     /**
      * @returns {Component[]}
```

Comment:
jsdoc makes no sense with ts. all jsdocs should be replaced with appropriate ts declarations